ext/import-font.h

```diff
@@ -23,6 +25,8 @@ bool getFontScale(double &output, FontHandle *font);
 bool getFontWhitespaceWidth(double &spaceAdvance, double &tabAdvance, FontHandle *font);
 /// Loads the shape prototype of a glyph from font file
 bool loadGlyph(Shape &output, FontHandle *font, int unicode, double *advance = NULL);
+/// Loads the spahe prototype of a glyph directly from FT glyph slot
+bool loadGlyphGlyph(Shape &output, FT_GlyphSlot glyph, double *advance = NULL);
```

Comment:
It probably makes more sense rename the old function to loadFontGlyph() or something, and let loadGlyph() be the name for your new version. It's a slightly larger change, but ultimately less confusing in the long run I'd wager.

Reply:
I would not think this justifiable enough to break api compatibility.
Anyway, since this is actually a c++ api, the new function could be loadGlyph too?

Reply:
Well, there isn't really an API compatibility guarantee in place, which is why I'm okay breaking things a bit. ;) But let's compromise and call it loadGlyphSlot()? I generally try and avoid overloaded functions in C++ for clarity.